stingray/powerspectrum.py

```diff
@@ -99,9 +107,9 @@ def _pavnosigfun(power, nspec):
     return sum
 
 
-class Powerspectrum(object):
+class Powerspectrum(Crossspectrum, object):
 
```

Comment:
Probably not. I think I learned it as good style somewhere, but that might be outdated.